src/connect/wrapMapToProps.ts

```diff
@@ -1,7 +1,16 @@
+import { Dispatch } from 'redux'
+
 import verifyPlainObject from '../utils/verifyPlainObject'
 
-export function wrapMapToPropsConstant(getConstant) {
-  return function initConstantSelector(dispatch, options) {
+export type MapToProps = {
+  (stateOrDispatch: any, ownProps?: any): any
```

Comment:
:hand: Let's replace all uses of `any` with either `type $FixTypeLater = any`, or `unknown`. I want it to be clear these are placeholders and not final